spec/controllers/surveys_controller_spec.rb

```diff
@@ -73,4 +72,122 @@
       expect(response.status).to eq(200)
     end
   end
+
+  describe "POST #create" do
```

Comment:
Nit: single quotes and below (or we can change to single quotes for the whole file later)